Add setter for release time

kumar-ish opened the pull request:
# Motivation
[filling in later]

# Changes
- add function to set release time

kumar-ish committed:
Add setter for release time

CalabashSquash commented on contracts/Receipt.sol:
```diff
@@ -37,6 +37,13 @@ contract Receipt is IReceipt, Ownable {
         transferOwnership(gov);
     }
 
+    /**
+     * @param newReleaseTime new release time
+     */
+    function setReleaseTime(uint256 newReleaseTime) external onlyOwner {
+        releaseTime = newReleaseTime;
+    }
+
```
Setters should be towards the bottom. E.g. check `setMaxSlippage` in this file.

sporejack replied:
Consult 797bb79.

sporejack committed:
Move release time setter to bottom of `Receipt.sol`

This commit moves `setReleaseTime` towards the bottom of the
`Receipt.sol` contract for styling reasons.

sporejack committed:
Rename release time setter parameter

This commit renames the parameter to `setReleaseTime` to align with
styling conventions.

sporejack committed:
Improve NatSpec comment for release time setter

sporejack committed:
Make  a call
